### DISCUSSION TITLE
Inter bold is too bold for emails clients. Any way to manipulate email clients to use a thinner variant as bold?

### DISCUSSION THREAD

**Discussion by dwabs** [rsms/inter#596] [Q&A]
We want to use Inter on email campaigns.

Turns out email clients only understand 2 variants of a font - Regular and Bold and there is no categorization for Medium/Semibold-like variants.

Inter Bold is just too bold on email clients (and renders badly in some) and I am wondering if it is possible for us to create a fork of the original Inter and fake the Medium or Semibold variant as the Bold variant for the font family instead?

We'll change the "Inter" name on the font to some custom name so that users with locally installed Inter don't get affected and their systems see this as a whole different font and loads the weights we supply.

Perhaps someone here has experience with this? Thanks in advance.

**Comment by kenmcd** [comment]
> Inter Bold is just too bold on email clients (and renders badly in some) and I am wondering if it is possible for us to create a fork of the original Inter and fake the Medium or Semibold variant as the Bold variant for the font family instead?
> 

Yes, it is really easy.
Using Inter v3.19? (the one currently in GF web fonts) Or?
What family name do you want?
And use the SemiBold as the Bold?  (Medium is too light)
And the Italics?
So you will end up with a custom R/I/B/BI font family.
Which you will need to host yourself.

**Reply by dwabs** [reply]
Using 3.19 yes
Family name can be anything, "Inter_mchrO3"
Yes, Semibold as Bold is better.
Regular Italic and Semibold Italic - yes
Yes, will be hosted.

Question is how we can generate such a fork with a custom family name?

**Reply by kenmcd** [reply]
There is not an easy way to do this via the repo.
Using the Inter v3.19 Windows Hinted fonts...
I renamed the fonts and fixed the style linking in TransType.
Converted the TTFs to WOFF/WOFF2 using ftCLI (in Python).
Try these: [Inter_mchrO3.v3.19.TTF.WOFF.WOFF2.zip](https://github.com/rsms/inter/files/12031007/Inter_mchrO3.v3.19.TTF.WOFF.WOFF2.zip)
Let me know if you have any issues.

**Reply by dwabs** [reply]
thanks. can you provide more instructions how this can be done manually (for other fonts)? I have TransType and Fontlab 8 both
